infra/batch/datasets/main.py

```diff
@@ -243,7 +303,6 @@ def batch_datasets(request):
     results = engine.execute(text(sql_statement)).all()
```

Comment:
suggestion: this method executes 1 + 1 query per feed resulting in +1300 queries. We can reduce the number of queries by unifying the query in line 301 with the one in line 310. The resulting query can look like:
```
select feed.stable_id, feed.producer_url,
gtfsfeed.id,
gtfsdataset.id as dataset_id, gtfsdataset.hash
from feed
join gtfsfeed on gtfsfeed.id=feed.id
left join gtfsdataset on gtfsdataset.feed_id = feed.id and gtfsdataset.latest = true
where
status='active' and authentication_type='0'
and (gtfsdataset.id is not NULL or gtfsdataset.latest is NULL)
```